fitbenchmarking/results_processing/performance_profiler.py

```diff
@@ -225,4 +279,143 @@ def create_plot(step_values: 'list[np.ndarray]', solvers: 'list[str]'):
                        )
             )
 
-    return fig
+    data_df = create_df(all_solvers,
+                        all_solver_values,
+                        all_plot_points)
```

Comment:
Yes, I think I see what you mean. This will need to change when we introduce the recalculation of the values. I left it simple for now because they are not being recalculated. But once we introduce that, we will need to save more values (or in other words bring more of the "raw" values along) as the function that recalculates the values to be plotted will be called directly by the callback. Does this answer your doubt? Or you meant something else?